docs/2_step_by_step/2_2_experiment_and_run.md

```diff
@@ -1 +1,33 @@
-# Two basic concepts: Experiment and Run
\ No newline at end of file
+# Two basic concepts: Experiment and Run
+
+Finetuner organizes your training based on two concepts: **Experiment** and **Run**.
+
+An Experiment is defined as a general purpose machine learning task you're fine-tuning for.
```

Comment:
``Experiment`` instead of Experiment?

Reply:
i think when using `Experiment`, it should be a piece of code. emphasis should use bold or it